### ISSUE TITLE
[투빅스]-빅데이터, 인공지능

### ISSUE THREAD

**Issue by danielgenius98** [danielgenius98/2019-1-teamproject#3]
참석자 정보
이준걸 / 대학원 1학년 / i201331749@gmail.com / 10기 회장
강인구 / 학부 4학년 / kangig94@gmail.com / 10기 부회장
동아리 소개
저희 동아리는 먼저 10주 동안 신입기수를 위한 머신러닝/딥러닝 집중교육이 있습니다. 이후에는 머신러닝/딥러닝/텍스트마이닝/강화학습 등 세부 세미나를 선택하여 심화교육을 이어갑니다. 또한 학기말에 있을 컨퍼런스를 위해 팀을 꾸려 프로젝트를 진행합니다. 그 외에도 정식 교육과정 외에 관심이 있고, 알아가고 싶은 부분이 있다면 스터디를 꾸리거나 참여해 공부할 수 있습니다.

투빅스 홈페이지 : http://www.datamarket.kr/
투빅스 소개 : https://drive.google.com/open?id=1ZND0dwRBTEF0HB8Uthh5wrK06tSr_3_U
투빅스 프로젝트 : http://www.datamarket.kr/xe/board_pdzw77
2018년 하반기 동아리 활동 및 수상 내역
- 활동내역
2014.1 투빅스 창립, 1기 활동 시작
2014.3 Datamarket 홈페이지 개설
2014.6 빅데이터 공개 특강 개최
2014.7 2기 활동 시작
2015.1 제1회 투빅스 데이터 분석 컨퍼런스 / 3기 활동 시작
2015.7 제2회 투빅스 데이터 분석 컨퍼런스 / 4기 활동 시작
2016.1 투빅스 X 와이빅타 연합 컨퍼런스 / 5기 활동 시작
2016.7 제3회 투빅스 데이터 분석 컨퍼런스 / 6기 활동 시작
2017.1 제4회 투빅스 데이터 분석 컨퍼런스 / 7기 활동 시작
2017.7 투빅스 X 와이빅타 연합 컨퍼런스 / 8기 활동 시작
2018.1 제5회 투빅스 데이터 분석 컨퍼런스 / 9기 활동 시작
2018.7 제6회 투빅스 데이터 분석 컨퍼런스 / 10기 활동시작
2019.1 제7회 투빅스 데이터 분석 컨퍼런스 / 11기 활동 시작

- 수상내역
<2015년 수상>

제1회 SKT BigData Analytics Festival 장려상
제2회 빅데이터 마케팅 분석보고서 공모전 대상
제3회 빅콘테스트 챌린지리그 미래창조과학부장관상
<2016년 수상>

제2회 L.Point Bigdata Competition 특별상
제1회 데이터시각화 대학생 Case Competition 우수상
신한은행 고객 아이디어 공모전 우수상
날씨 빅데이터 콘테스트 한국기상산업진흥원장 상
빅콘테스트 퓨쳐스리그 한국정보통신진흥협회 회장상
제 14회 SAS 분석 챔피언십 동상
제 14회 SAS 분석 챔피언십 HIRA상
<2017년 수상>

제 1회 미래에셋대우 빅데이터페스티벌 장려상
<2018년 수상>

빅콘테스트 챔피언 리그 한국정보화진흥원장 상
2회 미래에셋대우 빅데이터페스티벌 장려상
제7회 문화관광 빅데이터 분석 대회 은상
정보보호 R&D 데이터 챌린지 3등
제1회 우정사업본부 데이터사이언스 빅데이터 해커톤 우수상
2019 상반기 활동계획
신입기수 세미나
0주차 : R과 파이썬 예습하기
1주차 : 회귀분석 / 로지스틱 회귀 / Git
2주차 : 나이브베이즈 / SVM / KNN
3주차 : 의사결정나무 / 앙상블 / 클러스터링
4주차 : 차원축소 LDA / PCA
5주차 : Neural Network / 프레임워크
6주차 : 크롤링 / 텍스트 기초
7주차 : 텍스트 심화
8주차 : 이미지 기초
9주차 : 이미지 심화
10주차 : 강화학습
11주차 : 프로젝트 주제 발표

텍스트 세미나
Lecture 3 : GloVe: Global Vectors for Word Representation
Lecture 4: Word Window Classification and Neural Networks
Lecture 5: Backpropagation and Project Advice
Lecture 9: Machine Translation and Advanced Recurrent LSTMs and GRUs
Lecture 10: Neural Machine Translation and Models with Attention
Lecture 12: End-to-End Models for Speech Processing
Lecture 16: Dynamic Neural Networks for Question Answering
강화학습 세미나
intro
MDP
Planning by Dynamic Programming
Model Free Prediction
Model Free Control
Value Function Approximation
Policy Gradient Methods
Integrating Learning and Planning
Exploration and Exploitation
Case Study : Classic Games
머신러닝 세미나
회귀분석 심화
GAM
분석 프레임워크, NN's cost function
앙상블
San Francisco Crime Classification - 1 (Kaggle)
Google Analytics Customer Revenue Prediction (Kaggle)
function optimization and support vector machine
경제경영 데이터 분석(Dacon 3)
Recommender System
Structuring Machine Learning Projects
principle of classification(Bayes classifier&bayes error)
기타
투빅스 10기 회장 이준걸, 부회장 강인구 입니다. 잘부탁드립니다:)
